index.html

```diff
@@ -126,7 +127,35 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Laravel</a>
+                        <div class="cd-faq-content">
+                                <p>Eloquent (dostęp do bazy danych):</p>
+                                <ol>
+                                        <li>Dostępne są dwa query buildery: użycie <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php">pierwszego z nich</a> powoduje zwrócenie obiektów Active Record, natomiast <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php">drugi</a> zwraca tablice.</li>
```

Comment:
Norbercie, zapytania z query buildera ORMowego i DBALowego są zupełnie inne (a o takiej różnicy właśnie mówimy w przypadku Laravela - "ORM" i surowe zapytania). Zauważ też, że żaden z builderów doctrine'owych nie zwraca encji - encje może zwrócić co najwyżej repozytorium (w przypadku Laravela jest to odpowiedzialność query buildera - np. https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php#L75). Bardziej sensownie będzie chyba faktycznie położyć akcent w tym miejscu: zadaniem query buildera jest budowanie zapytań, a nie hydrowanie obiektów.